signbank/dictionary/templates/dictionary/gloss_revision_history_tab.html

```diff
@@ -2,5 +2,73 @@
 {% load bootstrap3 %}
 {% load i18n %}
 {% load static %}
+{% load comments %}
 
-REVISION HISTORY TAB
\ No newline at end of file
+<table class='table table-condensed'>
+    <tr>
+        <th>{% blocktrans %}Created:{% endblocktrans %}</th>
+        <td id='created'>{% if gloss.created_at %}<span class="glyphicon glyphicon-time" aria-hidden="true"></span> <em>{{ gloss.created_at|date:'Y-m-d H:i' }}</em> <span class="glyphicon glyphicon-user" aria-hidden="true"></span> {{ gloss.created_by.username }}{% endif %}</td>
+    </tr>
+    <tr>
+        <th>{% blocktrans %}Updated:{% endblocktrans %}</th>
+        <td id='updated'>{% if gloss.updated_at %}
+            <span class="glyphicon glyphicon-time" aria-hidden="true"></span> <em>{{ gloss.updated_at|date:'Y-m-d H:i' }}</em> <span class="glyphicon glyphicon-user" aria-hidden="true"></span> {{ gloss.updated_by.username }}
+            {% if request.user.is_staff %}
+            <!-- Modal trigger -->
+            <button type="button" class="btn btn-default" data-toggle="modal" data-target="#modalChangeHistory">
+              <span class="glyphicon glyphicon glyphicon-list-alt" aria-hidden="true"></span>
+                {% blocktrans %}Show complete history{% endblocktrans %}
+            </button>
+
+            <!-- Modal -->
+            <div class="modal fade" id="modalChangeHistory" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
```

Comment:
I think Micky was asking to just show the revision on this tab rather than in a modal? Have I misunderstood that?

Reply:
Just to put some notes in from the team standup today so we have a record:
The modal already exists, but the data to populate it appears not to, so it is coming up empty.
As per team standup I will now convert the modal popup code into something that sits properly in the tab, and double-check where the data source is for it.

Reply:
@jonholdsworth I think that we now have a story to move the revision history out the modal into a table and make sure it's tracked everywhere it should be. Since we've got a separate story for that, I'm happy to merge this as-is.

Reply:
No problem! I'll merge this one now!

Reply:
But I'll check the user auth first.